Author: Andreas Beckmann <debian@abeckmann.de>
Description: avoid linking the kernel module with ld.gold
 ld.gold does e.g. eliminate duplicate strings and this seems to cause
 problems with the nvidia kernel module.  Therefore explicitly link with
 ld.bfd.  This patch is not compatible with lenny which does neither have
 ld.bfd nor ld.gold, but we are not supporting lenny backports anyway.
Bug-Debian: http://bugs.debian.org/626279

--- a/Makefile
+++ b/Makefile
@@ -77,6 +77,8 @@
   endif
 
   .PHONY: modules module clean clean_conftest modules_install
+  # avoid ld.gold, see #626279
+  LD=ld.bfd
   modules clean modules_install:
 	$(MAKE) "CC=$(CC)" $(KBUILD_PARAMS) $@
 	@if [ "$@" = "modules" ]; then \
